public/css/lists.less

```
@@ -135,7 +135,7 @@
   padding-right: 1em;
 }
 
-.common .list-item {
+:not(.downtime-list).common .list-item {
   .caption {
```

I think this can be solved without an exclusion, but rather with an inclusion. It's only plugin output after all that's supposed to have multiple lines. It also solves the issue that plain comments are still not ellipsized.

```suggestion
.common .list-item {
  .caption .plugin-output {
```